sinonjs/referee-sinon
# Security: upgrade lint-staged

This removes the cryptiles dependency, which has a security advisory.

See https://nvd.nist.gov/vuln/detail/CVE-2018-1000620

#### How to issue

1. `git checkout 7fb69ecdfe5a2f113da3a0df44c6f11dbf105c34` (latest `master`).1
1. `npm ci`
1. `npm ls cryptiles`
1. Observe that `cryptiles@3.1.2` is in the output

#### How to verify - mandatory
1. Check out this branch
1. `npm ci`
1. `npm ls`
1. Observe that `cryptiles` is not in output

#### Checklist for author
